src/components/rmrk/Gallery/Collections.vue

```diff
@@ -0,0 +1,428 @@
+<template>
+  <div class="collections container">
+    <Loader :value="isLoading" />
+    <!-- TODO: Make it work with graphql -->
+    <b-field class="column">
+      <Pagination simple :total="total" v-model="currentValue" :perPage="perPage" replace class="is-right" />
+    </b-field>
+
+    <div>
+      <div class="columns is-multiline">
```

Comment:
we have `GalleryCard`

Reply:
GalleryCard is using only for NFT, not Collection.